src/express.re

```diff
@@ -268,7 +268,7 @@ module App = {
   external asMiddleware : t => Middleware.t = "%identity";
   /** [asMiddleware app] casts an App instance to a Middleware type */
 
-  external listen : t => port::int => unit = "" [@@bs.send];
+  external listen : t => port::int => onListen::(Js.Null_undefined.t Js.Exn.t => unit) [@bs.uncurry] => unit = "" [@@bs.send];
```

Comment:
default arguments would be neater. I updated the branch with your suggested changes.